src/GenericCharacter.jl

```diff
@@ -277,12 +277,16 @@ Return the generic character where the parameter `var` is replaced with `expr` i
 ```jldoctest
 julia> g=genchartab("GL2");
 
-julia> printval(g,char=1)
-Value of character type 1 on class type
-  1: exp(2π𝑖((2*i*k)//(q - 1)))
-  2: exp(2π𝑖((2*i*k)//(q - 1)))
-  3: exp(2π𝑖((i*k + j*k)//(q - 1)))
-  4: exp(2π𝑖((i*k)//(q - 1)))
+julia> g[1]
```

Comment:
I think we should add the ability to iterate over generic characters as well. Then this would simply be:
```suggestion
julia> collect(g[1])
```
And we would only get the values.

Reply:
If you think so too I can make a follow up PR addressing this.

Reply:
Yes that sounds like a good idea